website/docs/components/execution-context.md

```
@@ -0,0 +1,161 @@
+#### `IReadableContext`
+
+- `contains(token, matchValue)` — Checks if an array context value contains a specific item or matches a predicate.
+- `exists(token)` — Checks if a value exists for the token.
+- `missing(token)` — Checks if a value is missing for the token.
+- `get(token)` — Retrieves a value or null if not found.
+- `getOr(token, defaultValue)` — Retrieves a value or returns the provided default if not found.
+- `getOrFail(token)` — Retrieves a value or throws if not found.
+
+## Further information
+
+For further information refer to [`@daiso-tech/core/execution-context`](https://daiso-tech.github.io/daiso-core/modules/ExecutionContext.html) API docs.
```

markdownlint-cli2 (0.22.0)

[warning] Files should end with a single newline character

(MD047, single-trailing-newline)